jobrunner/cli/kill_job.py

```diff
@@ -15,12 +15,12 @@ def main(partial_job_ids, cleanup=False):
         # If the job has been previously killed we don't want to overwrite the
         # timestamps here
         if job.state in (State.PENDING, State.RUNNING):
-            mark_job_as_failed(job, "Killed by admin")
+            mark_job_as_failed(job, StatusCode.KILLED_BY_ADMIN)
```

Comment:
```suggestion
            mark_job_as_failed(job, StatusCode.KILLED_BY_ADMIN, "An OpenSAFELY admin manually killed this job", error=True)
```

I've realised that mark_job_as_failed should implicitly set error=True, but it deson't atm, so you'll need to do this to get an accurate error state.

And we should always give the user a helpful english message